### PR TITLE
Avatar Placeholder

### PR DESCRIPTION
**Pull request by martinezguillaume**
avr -05-2018 22-02-51

Add `placeholderStyle` and `PlaceholderContent` props

Avatar props from the gif above:
```js
{
  title: l.name[0],
  rounded: true,
  source: { uri: l.avatar_url },
}
```

If you want to customize the placeholder:
```js
{
  PlaceholderContent: <Text>salld</Text>,
  placeholderStyle: { backgroundColor: 'green' },
  title: l.name[0],
  rounded: true,
  source: { uri: l.avatar_url },
}
```

ref: https://github.com/react-native-training/react-native-elements/issues/336

### CODE REVIEW

**Comment by martinezguillaume** [comment]
I renamed renderPlaceholderContent to PlaceholderContent because we want a directly a React node

**Comment by iRoachie** [comment]
Alright that's merged, just need to resolve the conflicts

File: docs/avatar.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
----
-id: avatar
-title: Avatar
----
+------  ------
+id:     avatar
+title:  Avatar
+------  ------
```
Comment (iRoachie) [inline docs/avatar.md]: Need to fix this header. It'll break the website build process. It should be:

```
---
id: avatar
title: Avatar
---
```

**Comment by martinezguillaume** [comment]
Closed in favor of https://github.com/react-native-training/react-native-elements/pull/1211

**Pull request closed without merging**